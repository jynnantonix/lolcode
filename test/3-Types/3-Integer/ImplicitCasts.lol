This test checks that other types are implicitly cast to integers correctly.

HAI
	I HAS A var1 ITZ FAIL
	I HAS A var2 ITZ WIN
	I HAS A var3 ITZ "123"
	VISIBLE SUM OF 0 AN var1 MKAY?
	VISIBLE SUM OF 0 AN var2 MKAY?
	VISIBLE SUM OF 0 AN var3 MKAY?
KTHXBAI
