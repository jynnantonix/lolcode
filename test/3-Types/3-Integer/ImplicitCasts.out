0 
1 
Unexpected error: unsupported operand type(s) for +: 'int' and 'str'
